packages/widgets-internal/swap-v2/CurrencyInputPanelSimplify.tsx

```diff
@@ -0,0 +1,81 @@
+import { AtomBox, SwapCSS } from "@pancakeswap/uikit";
+
+import { NumericalInput, NumericalInputProps } from "./NumericalInput";
+
+interface CurrencyInputPanelProps extends Omit<NumericalInputProps, "onBlur" | "OnFocus"> {
+  onInputBlur?: () => void;
+  onInputFocus?: () => void;
+  id: string;
+  top?: React.ReactNode;
+  bottom?: React.ReactNode;
+  inputLeft?: React.ReactNode;
+  showBridgeWarning?: boolean;
```

Comment:
what's this property for?

Reply:
It seems related to the current Aptos swap input panel; if we don't need to apply this swap-revamp to Aptos in the future, then we don't need this for now